### DISCUSSION TITLE
【功能建议】增加TiDB支持

### DISCUSSION THREAD

**Discussion by jimsoft1000** [hhyo/Archery#2520] [Ideas]
现在TiDB数据库被很多公司在使用，也兼容mysql数据库的语法，建议在archery平台增加tidb数据库的支持

**Comment by LeoQuote** [comment]
你好, 本项目是一个开源项目, 你可以自由提功能需求, 但其实比提功能需求更加务实, 有价值的是实现这个功能, 本项目开发文档如下, 希望对你有帮助:

* https://github.com/hhyo/Archery/wiki/development
* https://github.com/hhyo/Archery/wiki/notifier

期待你的pr, 谢谢~

**Reply by jimsoft1000** [reply]
好的，文档写的非常好，我尝试开发一下
